MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/dds/RevealedPolygonHoleOutlineActivity.java

```diff
@@ -0,0 +1,205 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import timber.log.Timber;
+
+import static com.mapbox.mapboxsdk.style.layers.PropertyFactory.fillColor;
+import static com.mapbox.mapboxsdk.style.layers.PropertyFactory.fillOpacity;
+import static com.mapbox.mapboxsdk.style.layers.PropertyFactory.lineCap;
+import static com.mapbox.mapboxsdk.style.layers.PropertyFactory.lineColor;
+import static com.mapbox.mapboxsdk.style.layers.PropertyFactory.lineJoin;
+import static com.mapbox.mapboxsdk.style.layers.PropertyFactory.lineWidth;
+
+public class RevealedPolygonHoleOutlineActivity extends AppCompatActivity implements OnMapReadyCallback {
+
+  private static final String FILL_SOURCE_ID = "SOURCE-ID";
+  private static final String LINE_SOURCE_ID = "LINE_SOURCE_ID";
+  private static final String FILL_LAYER_ID = "FILL-LAYER-ID";
+  private static final String LINE_LAYER_ID = "LINE-LAYER-ID";
+  private static final float FILL_OPACITY = .55f;
+  private static final float LINE_WIDTH = 3f;
+  private static final int GREY_COLOR = Color.parseColor("#c2c2c2");
+  private static final int RED_COLOR = Color.parseColor("#BF544C");
+  private MapView mapView;
+  private MapboxMap mapboxMap;
+  private static final List<Point> OUTER_POLYGON_COORDINATES = new ArrayList<Point>() {
+    {
+      add(Point.fromLngLat(-121.9921875, 37.27787748952485));
+      add(Point.fromLngLat(-121.79580688476562, 37.27787748952485));
+      add(Point.fromLngLat(-121.79580688476562, 37.40452830389465));
+      add(Point.fromLngLat(-121.9921875, 37.40452830389465));
+      add(Point.fromLngLat(-121.9921875, 37.27787748952485));
+    }
+  };
+
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+
+    // Mapbox access token is configured here. This needs to be called either in your application
+    // object or in the same activity which contains the mapview.
+    Mapbox.getInstance(this, getString(R.string.access_token));
+
+    // This contains the MapView in XML and needs to be called after the access token is configured.
+    setContentView(R.layout.activity_polygon_holes_outline_revealed);
+
+    mapView = findViewById(R.id.mapView);
+    mapView.onCreate(savedInstanceState);
+    mapView.getMapAsync(this);
+  }
+
+  @Override
+  public void onMapReady(final MapboxMap map) {
+    this.mapboxMap = map;
+    map.setStyle(Style.MAPBOX_STREETS, new Style.OnStyleLoaded() {
+      @Override
+      public void onStyleLoaded(@NonNull Style style) {
+        new LoadGeoJson(RevealedPolygonHoleOutlineActivity.this).execute();
+      }
+    });
+  }
+
+  private void addStyling(@NonNull List<Point> pointList) {
+    if (mapboxMap.getStyle() != null) {
+      Style loadedStyle = mapboxMap.getStyle();
+
+      List<LineString> innerList = new ArrayList<>();
+      innerList.add(LineString.fromLngLats(pointList));
+
+      List<List<Point>> megaInnerList = new ArrayList<>();
```

Comment:
This list seems obsolete if you can use `LineString` as a geometry for the `LineLayer`.

Reply:
Removed